Client/mods/deathmatch/logic/CClientObjectManager.h

```diff
@@ -25,6 +25,7 @@ class CClientObjectManager
 
     void DeleteAll();
 
+    const auto&           GetObjects() const { return m_Objects; };
```

Comment:
```suggestion
    const CMappedArray<CClientObject*>& GetObjects() const { return m_Objects; };
```
The return type is `CMappedArray<CClientObject*>&` It doesn't take that much space, I don't think the use of auto keyword is justified here.